server/bot/imports/tools.ts

```diff
@@ -95,3 +95,7 @@ export const fetchLimited = async (url: string, limit: number, opts = {}): Promi
     req.on('error', reject)
   })
 }
+
```

Comment:
Either replace the other occurrences of this code with `sleep` calls, or else do away with this function and just use `await new Promise(resolve => setTimeout(resolve, 1000))` instead.